### ISSUE TITLE
Failing unit test check_pubsub_encoding_custom

### ISSUE THREAD

**Issue by keba-estr** [open62541/open62541#5707]
## Description

Since PR #5681 the new unit test check_pubsub_encoding_custom fails (sporadically?) with some heap-buffer-overflow message.

Example build: https://github.com/open62541/open62541/actions/runs/4489318805/jobs/7894990967?pr=5697

@basyskom-dege Pipeline fails from first uncritical commit on -> problem because of PR #5681, please have a look at it.

## Background Information / Reproduction Steps

Used CMake options:

```bash
cmake -DUA_NAMESPACE_ZERO=<YOUR_OPTION> <ANY_OTHER_OPTIONS> ..
```

## Checklist

Please provide the following information:

 - [ ] open62541 Version (release number or git tag):
 - [ ] Other OPC UA SDKs used (client or server): 
 - [ ] Operating system:
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

**Comment by jpfr** [comment]
I ran into similar issues and fixed them as part of #5693.
Please verify if this is resolved and close.

**Comment by keba-estr** [comment]
Problem does not occur any more after update to new commits.
